### ISSUE TITLE
[ci][release]: Fix 3.x backport git requirements

### ISSUE THREAD

**Issue by v1v** [elastic/apm-agent-ruby#796]
```
12:51:00  [Pipeline] sh
12:51:01  + rake release:update_branch
12:51:01  error: pathspec '3.x' did not match any file(s) known to git
12:51:01  hint: '3.x' matched more than one remote tracking branch.
12:51:01  hint: We found 2 remotes with a reference that matched. So we fell back
12:51:01  hint: on trying to resolve the argument as a path, but failed there too!
12:51:01  hint: 
12:51:01  hint: If you meant to check out a remote tracking branch on, e.g. 'origin',
12:51:01  hint: you can do so by fully qualifying the name with the --track option:
12:51:01  hint: 
12:51:01  hint:     git checkout --track origin/<name>
12:51:01  hint: 
12:51:01  hint: If you'd like to always have checkouts of an ambiguous <name> prefer
12:51:01  hint: one remote, e.g. the 'origin' remote, consider setting
12:51:01  hint: checkout.defaultRemote=origin in your config.
12:51:01  [Pipeline] }
```

**Comment by v1v** [comment]
As soon as we release a new version of the library, the issue should be fixed.

**Comment by v1v** [comment]
Released on [v1.1.100](https://apm-ci.elastic.co/job/apm-shared/job/apm-pipeline-library-mbp/view/tags/job/v1.1.100/)
